src/js/models/maps/assets/MapAsset.js

```diff
@@ -142,6 +146,9 @@ define([
        * opacity of the layer on the map, with 0 representing fully transparent and 1
        * representing fully opaque. This applies to raster (imagery) and vector assets,
        * not to terrain assets.
+       * @property {Number} [saturation=1] - A number between 0 and 1 indicating the
```

Comment:
You can also set it to any number > 1 to increase saturation beyond the image's current level

Reply:
Indeed! I also realized negative number works as well, though it's not what I expected... I just copied over Cesium's documentation. https://cesium.com/learn/ion-sdk/ref-doc/ImageryLayer.html#saturation